addons/wp/stringtable.xml

```diff
@@ -170,7 +170,7 @@
             <Polish>Zadanie - Atak/Odwrót</Polish>
             <Russian>Задача Нападение/Отступление</Russian>
             <Italian>Compito Assalto/Ritirata</Italian>
-            <Chinesesimp>撤退/突袭任务</Chinesesimp>
+            <Chinesesimp突袭/撤退任务</Chinesesimp>
```

Comment:
> missing here
```suggestion
            <Chinesesimp>突袭/撤退任务</Chinesesimp>
```

Reply:
Sorry, I accidentally deleted something when copying files. Thank you for your correction